src/modules/launcher/Plugins/Microsoft.Plugin.Calculator/CalculateEngine.cs

```diff
@@ -3,14 +3,21 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Generic;
 using System.Globalization;
 using Mages.Core;
 
 namespace Microsoft.Plugin.Calculator
 {
     public class CalculateEngine
     {
-        private readonly Engine _magesEngine = new Engine();
+        private readonly Engine _magesEngine = new Engine(new Configuration
+        {
+            Scope = new Dictionary<string, object>
+            {
+                { "e", Math.E } // e is not contained in the default mages engine
+            }
+        });
```

Comment:
The compiler complains about a missing blank line after this one, and a missing colon after `}` on lines 18 and 19.

Reply:
Do you mean a missing comma, so like that?
```
private readonly Engine _magesEngine = new Engine(new Configuration
{
    Scope = new Dictionary<string, object>
    {
        { "e", Math.E }, // e is not contained in the default mages engine
    },
});


public const int RoundingDigits = 10;
```

Reply:
Exactly!